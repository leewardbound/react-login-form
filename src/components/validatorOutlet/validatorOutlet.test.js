import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import ValidatorOutlet from '.';

const renderer = new ShallowRenderer();

describe('ValidatorOutlet', function () {
  it('should render correctly', function () {
    const component = renderer.render(
      <ValidatorOutlet>Lorem ipsum</ValidatorOutlet>
    );

    expect(component).toMatchSnapshot();
  });
});
