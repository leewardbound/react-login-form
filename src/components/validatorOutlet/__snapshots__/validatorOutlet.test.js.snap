// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ValidatorOutlet should render correctly 1`] = `
<span
  className="guGSmZ"
>
  Lorem ipsum
</span>
`;
